src/templates/next/layouts/render.tsx

```diff
@@ -103,5 +103,9 @@ export const renderLayout = (props: IsomerPageSchema) => {
       return <NotFoundLayout {...props} />
     case "search":
       return <SearchLayout {...props} />
+    // These are references that we should not render to the user
+    case "file":
+    case "link":
+      return <></>
```

Comment:
was a bit confused by this, took some time to think about it
Is the reason why we need a new `layout` for `file` and `article` because we want them to be stored as their own .json file?
I wonder if its cleaner to just update the `items` in the collection layout json

can discuss more on a call if needed

Reply:
Yup, I felt that they should be stored as a separate json file so that there is a "physical" representation of each item in a collection. I avoided using the `items` prop here because that might mean some items in the collection being stored in a single file, whereas other items is stored as separate files.